Signal/src/view controllers/SignalAttachment.swift

```diff
@@ -97,7 +97,25 @@ class SignalAttachment: NSObject {
             assert(false)
             return nil
         }
-        return "\(error)"
```

Comment:
errorDescription used to be used for logging only.  Now it's used for logging and for user-facing error messages. We could separate these out into two separate methods...